DESCRIPTION

```diff
@@ -12,15 +12,17 @@ Imports:
     tibble,
     tidyr,
     purrr,
-    magrittr
+    magrittr,
+    pkgload
```

Comment:
I prefer alphabetic ordering of dependencies. Don't need to change anything now, just keep in mind for the next time we edit here.